synthetics/README.md

```
@@ -0,0 +1,14 @@
+# qoodish-web-synthetics
+
+A Cloudflare Worker that checks production every 30 minutes with a real browser through Browser Rendering. Each run verifies `/api/health`, the top page, and a public map detail page on the `TARGET_ORIGIN` configured in `wrangler.jsonc`, and fails the scheduled invocation when any check does, so failures are visible in the Cloudflare dashboard.
+
+## Alerts
+
+Failures are also posted to the webhook stored in the `ALERT_WEBHOOK_URL` secret. Slack incoming webhooks and Discord webhooks both work; without the secret, failures are only visible in the dashboard.
+
+## Deploy
+
+```bash
+$ pnpm wrangler secret put ALERT_WEBHOOK_URL
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)